src/KingSystem/World/worldShootingStarMgr.cpp

```diff
@@ -30,27 +30,32 @@ void ShootingStarMgr::init_(sead::Heap* heap) {
         initSchedule();
 }
 
-// NON_MATCHING
 void ShootingStarMgr::initSchedule() {
-    if (sHours.start >= sHours.end) {
-        sead::FixedObjArray<u32, 24> validHours;
+    const s32& start_hour = sHours.start;
```

Comment:
Does this match without the reference?

Reply:
It doesn't. `start_hour` has to be a reference here. But `end_hour` matches with or without the reference.

Reply:
But it wouldn't make much sense for `end_hour` to be a reference here, since it's computed.

Reply:
That's unfortunate but not that strange so I think it's fine for start_hour to be a reference.